templates/patterns/progress/progress.ui_patterns.yml

```diff
@@ -0,0 +1,49 @@
+progress:
+  label: "Progress"
+  description: "Indicate the progress as a bar. https://getbootstrap.com/docs/5.0/components/progress/"
+  settings:
+    style:
+      type: "select"
+      label: "Bar style"
+      options:
+        primary: "primary"
+        secondary: "secondary"
+        success: "success"
+        danger: "danger"
+        warning: "warning"
+        info: "info"
+        light: "light"
+        dark: "dark"
+      preview: "primary"
+    striped:
+      type: "boolean"
+      label: "Striped style of state"
+      preview: true
+    animated:
+      type: "boolean"
+      label: "Animation on striped style of state"
+      preview: true
+  fields:
+    label:
+      type: "text"
+      label: "Label in the bar"
+      preview: "This is the label of progress bar"
+    progress:
+      type: "text"
+      label: "Percentage of progress in the bar"
+      preview: "50"
+    bar_label:
+      type: "text"
+      label: "Text indicating the percentage of progress in the bar"
+      preview: "50%"
+    message:
+      type: "text"
+      label: "Optional description text below the progress bar"
+    min:
+      type: "text"
+      label: "Minimum state of prorgress bar"
+      preview: "0"
+    max:
```

Comment:
Same for max, should be placed at settings.

Reply:
Min and Max are fields due to free text is not allowed on the settings section and at alexis branch this were also fields, not settings.